src/apps/run/components/juttle-viewer.js

```diff
@@ -15,10 +15,14 @@ class JuttleViewer extends Component {
     }
 
     render() {
-        let juttleSource = false;
```

Comment:
juttleSource seemed like a better variable name than the generic "bundle", can we keep it?